README.md

```diff
@@ -153,8 +153,9 @@ created for you automatically.
 
 ## Complete Production Installation Documentation:
 
-- [OpenShift Container Platform](https://docs.openshift.com/container-platform/3.11/install/running_install.html)
-- [OpenShift Origin](https://docs.okd.io/latest/install/index.html)
+- [OpenShift Container Platform](https://docs.openshift.com/container-platform/3.11/install/index.html)
+- [OKD](https://docs.okd.io/3.11/install/index.html) (formerly OpenShift Origin)
```

Comment:
Origin is active name though, please revert this part

Reply:
Shouldn't the name OKD be included here? Perhaps the issue is more with the word `formerly`?

Reply:
As far as I understood it, the upstream project was renamed from[ "OpenShift Origin" to "OKD"](https://blog.openshift.com/okd310release/) with the release of 3.10. Of course "Origin" is still part of the new name. But "OpenShift" itself isn't anymore. I think the link should be called OKD, but please let me just know the name of the link and I'll update the PRs and correct those that have already been merged.